classes/collection/BrandCollection.php

```diff
@@ -47,7 +48,16 @@ public function active()
      */
     public function category($iCategoryID)
     {
-        $arResultIDList = BrandListStore::instance()->category->get($iCategoryID);
+        $obCategoryItem = CategoryItem::make($iCategoryID);
+        $arResultIDList = [];
+
+        if ($obCategoryItem->children->isNotEmpty()) {
```

Comment:
The method does not take into account nesting of categories over 1 level

Reply:
Fixed